README.md

```diff
@@ -321,9 +324,9 @@ folder_permission = 0o0700
 #kv_version=1
 ```
 
-Additionally, environment variables `VAULT_ADDR` and `VAULT_TOKEN` are bound to `global.address` and `global.token`
-configuration variables for convenience, since they might be already in place under a shell using vault cli.
-As a last option, token can be retrieved from the `~/.vault-token` helper file.
+Additionally, environment variables `VAULT_ADDR`, `VAULT_TOKEN` and `VUALT_NAMESPACE` are bound to `global.address`,
```

Comment:
```suggestion
Additionally, environment variables `VAULT_ADDR`, `VAULT_TOKEN` and `VAULT_NAMESPACE` are bound to `global.address`,
```

Reply:
I see 2 modified files:

* `README.md` (so documentation)
* `root.go` , that allows the new property `global.namespace`  to be set either via an argument (`rvault -n <namespace>`), an environment variable `VAULT_NAMESPACE`  or through the configuration file.

But that property is not read or used anywhere else in the code. How did you test it?

I´m wondering if there is some under the hoods logic in vault API module that will use namespaces if `VAULT_NAMESPACE` is set in your shell (so actually passing `-n <namespace>` or having it in the config file doesn´t matter, it´s just a matter of that env var being present?